### PR TITLE
Feature signup testing

### PR DESCRIPTION
**Pull request by ema999**
## Summary

I have added the test file for singup component.

## Trello Card

https://trello.com/c/UygMjeDQ

### CODE REVIEW

File: src/components/SignupForm/SignupForm.test.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```
Comment (alejobh) [inline src/components/SignupForm/SignupForm.test.js]: rename this file to `index.test.js`

**Review by alejobh — COMMENTED** [review]
just one comment

File: src/components/SignupForm/index.test.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    fireEvent.change(inputPassword, { target: { value: 'short' } });
+    fireEvent.change(inputConfirmPassword, { target: { value: 'short' } });
+
+    fireEvent.click(signUpButton);
+
+    const error = await waitFor(() => screen.getByTestId('error'));
+    expect(error).toBeInTheDocument();
+  });
+
+  test('The email is incorrect', async () => {
```
Comment (DavidRojas1612) [inline src/components/SignupForm/index.test.js:61]: Are you sure to need to validate all fields in this test?

**Review by DavidRojas1612 — COMMENTED** [review]
Nice job! in the registration tests I recommend that you add a test case for the passwords

**Review by DavidRojas1612 — APPROVED** [review]

**Review by alejobh — APPROVED** [review]

**Pull request merged**